config/helpers.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../database/dbConfig')
+
+module.exports = {
+  findUser,
+  addUser
+}
+function findUser(user) {
+  return db('users')
+    .where({ username: user.username })
```

Comment:
You could [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) off of the user parameter to shorten this:
```javascript
unction findUser({ username }) {
  return db('users')
    .where({ username })
    .first()
```